src/Routes.php

```diff
@@ -24,7 +24,30 @@ class Routes extends AbstractEndpoint {
 	 * @return array|\WP_Error
 	 */
 	public function endpoint_callback( \WP_REST_Request $request ) {
-		$date = [];
+		$data = [];
+
+		// Create a route for each page.
+		$pages_query = new \WP_Query([
+			'post_type' => 'page',
```

Comment:
If I'm not wrong this is only going to return a limited number of pages instead of all of them.